web-ui/src/pages/PermissionsPage.jsx

```diff
@@ -83,7 +83,7 @@ const EditPermissionsPage = () => {
         setSelectedRole(role);
       },
       toQP(selectedRole) {
-        return selectedRole ? selectedRole.role : '';
+        return selectedRole?.role ?? '';
```

Comment:
If this doesn't work, a `?? { role: '' }` may be what you're looking for. It's difficult to tell at eyeshot without types.

Reply:
In this case the return value is used as a query parameter value, so it needs to be a string.